Update biosig.yaml

mandian opened the pull request:
Hi,

biosig has been updated to 2.5.x a while ago but Octave PAckages is still pointing to version 2.4.3.

siko1056 commented:
Hello @mandian ,

Thank you for your new releases.  I made two smaller changes to your pull request, but still the `doctest` is failing with the following output:
```
eeglab/
      eegplugin_biosig.m ................................. FAIL    1/1   
  
     >> eegplugin_biosig(fig, trystrs, catchstrs);
  
        expected: 
   Inputs:
     fig        - [integer]  EEGLAB figure
     trystrs    - [struct] "try" strings for menu callbacks.
     catchstrs  - [struct] "catch" strings for menu callbacks. 
  
   Create a plugin:
     For more information on how to create an EEGLAB plugin see the
     help message of eegplugin_besa() or visit http://www.sccn.ucsd.edu/eeglab/contrib.html
  
   Author: Arnaud Delorme, SCCN/INC, 25 Nov. 2003
  
        got     : 'fig' undefined near line 1, column 18
  
      pop_biosig.m ....................................... FAIL    2/2   
  
     >> OUTEEG = pop_biosig; % pop up window
  
        expected: 
        got     : requires the EEGLAB toolbox for pop up window
  
     >> OUTEEG = pop_biosig( filename, channels, type);
  
        expected: 
   Inputs:
     filename - [string] file name
     channels - [integer array] list of channel indices
     type     - [string] file type. See sload() function.
  
   Outputs:
     OUTEEG   - EEGLAB data structure
  
   Author: Arnaud Delorme, SCCN, INC, UCSD, Oct. 29, 2003
  
   Note: BIOSIG toolbox must be installed. Download BIOSIG at 
         http://sourceforge.net/project/showfiles.php?group_id=7072
         (please let us know at eeglab@sccn.ucsd.edu if this link is outdated).
         Contact alois.schloegl@gmail.com for troubleshooting using BIOSIG.
  
        got     : 'filename' undefined near line 1, column 22
  
      pop_readbdf.m ...................................... FAIL    3/3   
  
     >> EEG = pop_readbdf;             % an interactive window pops up
  
        expected: 
        got     : This function requires the EEGLAB toolbox
  
     >> EEG = pop_readbdf( filename ); % no pop-up window 
  
        expected: 
        got     : 'filename' undefined near line 1, column 20
  
     >> EEG = pop_readbdf( filename, range, eventchans, ref );
  
        expected: 
   Graphical interface:
     "Data block range to read" - {edit box] see command line 
                      parameter 'range' below.
     "Event channel index(s)" - {edit box] see command line 
                      parameter 'eventchans' below.
     "Index(s) of reference channel(s)" - {edit box] see command line 
                      parameter 'ref' below.
   Inputs:
     filename       - Biosemi 24-bit BDF file name
  
   Optional input:
     range          - [min max] integer range of data blocks to import.
                      Default is empty -> import all data blocks.
     eventchans     - [integer] event channel index(s). Default: none.
     ref            - [integer] channel index or index(s) for the reference.
                      Reference channels are not removed from the data,
                      allowing easy re-referencing. If more than one
                      channel, data are referenced to the average of the
                      indexed channels. WARNING! Biosemi Active II data 
                      are recorded reference-free, but LOSE 40 dB of SNR 
                      if no reference is used!. If you do not know which
                      channel to use, pick one and then re-reference after 
                      the channel locations are read in. {default: none}
   Outputs:
     EEG            - EEGLAB data structure
  
   Author: Arnaud Delorme, CNL / Salk Institute, 13 March [200](https://github.com/gnu-octave/packages/actions/runs/6750093110/job/18351807475?pr=357#step:7:207)2
  
   See also: openbdf(), readbdf()
  
        got     : 'filename' undefined near line 1, column 20
  
  Can't open display: 
      pop_readedf.m ...................................... 
  Error: Process completed with exit code 1.
```

Note that the package test suite is rather an automated sanity check.  If you are okay with the result and the failure is "expected" or can safely be ignored, please just merge this pull request at any time.

mandian commented:
Hello @siko1056,

I am not a biosig developer but I package octave and most of its package from https://gnu-octave.github.io/packages/ for OpenMandriva distribution.

I can't reproduce the error you showed, it look like you should install eeglab package first, but I can't reproduce it in OpenMandriva. Here is my output

```
> pkg test biosig
Te pkg test biosig
Testing functions in package 'biosig':

Integrated test scripts:

  ..ve/api-v58/packages/biosig-2.5.2/t300_FeatureExtraction/csp.m  pass    1/1   
  ..5.2/t300_FeatureExtraction/get_local_maxima_above_threshold.m 0 spurious events removed
1 spurious events removed
0 spurious events removed
1 spurious events removed
 pass   11/11  
  ..tave/api-v58/packages/biosig-2.5.2/t400_Classification/covm.m  pass    3/3   
  ..pi-v58/packages/biosig-2.5.2/t400_Classification/sumskipnan.m  pass    4/5   
                                                                   FAIL    1

Fixed test scripts:


Failure Summary:

  ..pi-v58/packages/biosig-2.5.2/t400_Classification/sumskipnan.m  pass    4/5   
                                                                   FAIL    1

Summary:

  PASS                               19
  FAIL                                1

See the file /home/mandian/abf/kicad/fntests.log for additional details.

sting functions in package 'biosig':

Integrated test scripts:

  ..ve/api-v58/packages/biosig-2.5.2/t300_FeatureExtraction/csp.m  pass    1/1   
  ..5.2/t300_FeatureExtraction/get_local_maxima_above_threshold.m 0 spurious events removed
1 spurious events removed
0 spurious events removed
1 spurious events removed
 pass   11/11  
  ..tave/api-v58/packages/biosig-2.5.2/t400_Classification/covm.m  pass    3/3   
  ..pi-v58/packages/biosig-2.5.2/t400_Classification/sumskipnan.m  pass    4/5   
                                                                   FAIL    1

Fixed test scripts:


Failure Summary:

  ..pi-v58/packages/biosig-2.5.2/t400_Classification/sumskipnan.m  pass    4/5   
                                                                   FAIL    1

Summary:

  PASS                               19
  FAIL                                1

See the file /home/mandian/abf/kicad/fntests.log for additional details.


```

That is due to nothing compiles sumskipnan_mex.cpp.

siko1056 commented:
Hi @mandian , 
Thank you for your reply.  It seems we ignored it in the previous release as well: https://github.com/gnu-octave/packages/pull/200#issuecomment-1281108237

I just went ahead with the release now.
